src/jit/emitxarch.cpp

```diff
@@ -3858,6 +3858,31 @@ void emitter::emitIns_R_R_R_I(
     emitCurIGsize += sz;
 }
 
+void emitter::emitIns_R_R_R_R(
+    instruction ins, emitAttr attr, regNumber targetReg, regNumber reg1, regNumber reg2, regNumber reg3)
```

Comment:
As a consequence of previous comment we need `emitIns_R_R_R_I` to support  `RWR_RRD_RRD_CNS`.